pkg/cmd/cli/cmd/login/logout.go

```diff
@@ -123,25 +126,40 @@ func (o LogoutOptions) RunLogout() error {
 		return err
 	}
 
-	if err := client.OAuthAccessTokens().Delete(token); err != nil {
-		return err
+	errs := []error{}
+	serverErr := client.OAuthAccessTokens().Delete(token)
+	if serverErr != nil {
```

Comment:
I need to not append `nil` to the `errs` slice at the end in order to print a successful message or not by checking `if len(errs) == 0 ...`